_posts/2023-01-11-install-and-manage-applications.md

```diff
@@ -11,555 +11,245 @@ published: true
 
 ## Introduction
 
-Vanilla OS allows installing packages and software from multiple sources. This guide will discuss them in detail.
+Vanilla OS allows installing packages and software from various sources. This guide will discuss them in detail.
 
-Vanilla OS comes with the option to enable Flatpak, AppImage and Snap in the initial setup. It allows the installation of native packages using the `apx` package manager. (**_Note_**: At the time of writing this guide, Snap is not yet supported in Vanilla OS)
+Vanilla OS comes with the option to enable Flatpak, AppImage and Snap in the initial setup. It also allows the installation of native packages using the `apx` package manager. (**_Note_**: At the time of writing this guide, Snap is not yet supported in Vanilla OS)
```

Comment:
```suggestion
Vanilla OS comes with the option to enable Flatpak, AppImage and Snap in the initial setup. It also allows the installation of native packages using the `apx` package manager. (**_Note_**: Snap is currently not supported in Vanilla OS)
```
In my opinion the `at the time of writing this guide` is not really useful in this case as it could give the impression that it may work when the user is reading the guide.